code/modules/mob/living/life.dm

```diff
@@ -152,4 +152,16 @@
 	var/grav_strength = gravity - GRAVITY_DAMAGE_THRESHOLD
 	adjustBruteLoss(min(GRAVITY_DAMAGE_SCALING * grav_strength, GRAVITY_DAMAGE_MAXIMUM) * seconds_per_tick)
 
+/mob/living/proc/currently_breathing()
```

Comment:
Docs

In particular you may want to specify what "breathing" means. Because something like a simple mob does not "breathe" (in that they don't take in / exhale air like a carbon) but they do "breathe" (flavor-wise).